common/utils/src/main/resources/error/error-conditions.json

```
@@ -1144,6 +1144,19 @@
     ],
     "sqlState" : "42623"
   },
+  "DESCRIBE_COL_JSON_UNSUPPORTED" : {
+    "message" : [
+      "DESCRIBE TABLE ... AS JSON not supported for individual columns."
+    ],
+    "sqlState" : "0A000"
+  },
+  "DESCRIBE_JSON_NOT_EXTENDED" : {
+    "message" : [
+      "DESCRIBE TABLE ... AS JSON only supported when [EXTENDED|FORMATTED] is specified.",
+      "For example: `DESCRIBE EXTENDED TABLE t AS JSON` is supported but `DESCRIBE TABLE t AS JSON` is not."
```

Let's follow to the current behaviour of `toSQLStmt`
```suggestion
      "For example: DESCRIBE EXTENDED TABLE t AS JSON is supported but DESCRIBE TABLE t AS JSON is not."
```